maps/unused/ozymandias.dmm

```diff
@@ -84653,7 +84653,7 @@
 /area/station/quartermaster/office)
 "vJl" = (
 /obj/machinery/manufacturer/general/grody{
-	free_resource_amt = 2
+	free_resources = list()
```

Comment:
very minor but this is a functional change (should have 2 of everything)

Reply:
Huh, didn't know. grody manufacturers clear their free_materials list on the type, so they shouldnt have anything really

Reply:
![image](https://github.com/goonstation/goonstation/assets/27376947/4632d30a-3c28-4b7e-94f1-a5650b327bb9)
it never has anything though. I can give it materials by default if this is desired

Reply:
(screenshot from main branch)

Reply:
Oh, whoops. In that case, just remove the line entirely? Since the varedit on the map isn't actually doing anything.

Reply:
Ah true, ha